src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -337,6 +337,82 @@ .maxstack  7
         }
 
         [Fact]
+        public void SimpleTupleNew2()
+        {
+            var source = @"
+class C
+{
+    static void Main()
+    {
+        var x = new (int a, int b)(1, 2);
```

Comment:
Turns out we did not support constructor initializers for tuples at all. 
Added positive/negative tests